scripts/lib.sh

```diff
@@ -20,13 +30,34 @@ query-mac() {
     [[ $(uname -s) = 'Darwin' ]]
 }
 
-query_big_sur() {
+query-big-sur() {
```

Comment:
I think if the return value is a boolean, we should rename to be `is-<object>`

Reply:
Seems like we have a mix of using `_` and `-` as separators. (e.g. in `.envrc`)

Reply:
Hmm I'd probably use it when generically interacting with a service or something along those lines. e.g. `query_google(search_term)`